pkg/virt-handler/vm.go

```diff
@@ -88,8 +90,11 @@ func NewController(
 	ipAddress string,
 	virtShareDir string,
 	virtPrivateDir string,
+	libvirtConnection *libvirt.Connect,
 	vmiSourceInformer cache.SharedIndexInformer,
 	vmiTargetInformer cache.SharedIndexInformer,
+	nodeInformer cache.SharedIndexInformer,
```

Comment:
I'm still puzzled about the need for this informer.
@ksimon1 Could you please explain the logic a bit more?

Reply:
If node gets updated (e.g. someone replaces all labels), labeller needs to check if all labeller's labels are ok.

Reply:
When the node gets updated, won't it also mean that virt-handler will be restarted?

Reply:
But if node gets updated only with new labels, it will not require restart

Reply:
I guess this is the part that I don't understand. :) That's why I keep asking... sorry about that.
Could you please show which labels could be updated, and what sets them?

Reply:
@davidvossel  Exactly that is what I had on my mind

Reply:
ah, got it. Thanks @davidvossel